debian/rules

```diff
@@ -11,7 +11,7 @@
 export VERBOSE=1
 
 %:
-	dh $@ --parallel --with autotools_dev,autoreconf,bash_completion,python3
```

Comment:
Why are we removing the autotools_dev and autoreconf addons here?

Reply:
Oof, it has been some months and now I don't remember. I would guess that they weren't necessary or caused some error? I will re-stest this and let you know.

Reply:
@yoe Aha! It was to match your version here, so I would assume they are not necessary:

https://salsa.debian.org/wouter/ola/-/blob/3f36e8f3bab30f4cf631f5503894049cedcba201/debian/rules

Reply:
*blush* okay, that explains that. I guess they're not needed anymore then :)